vispy/scene/cameras/cameras.py

```diff
@@ -699,11 +693,7 @@ def _update_transform(self):
         # side (as if z was flipped).
 
         if self.up == '+z':
-            # Keep it simple, but we need to pass AffineTransform,
-            # because if we toggle to -z and then back to +z, things
-            # work incorrectly (test with flipped_axis.py)
-            # todo: fix this so we can just set to a STTransform
-            thetransform = self.transform * AffineTransform()
+            thetransform = self.transform
```

Comment:
I'd prefer to leave as-is until we actually fix the problem, which will probably be automatically fixed as we move forward with the transform system.